relay-server/src/actors/upstream.rs

```diff
@@ -458,6 +483,53 @@ impl UpstreamRelay {
             .spawn(ctx);
     }
 
+    /// Adds a metric for the upstream request.
+    fn meter_result(
+        request: &UpstreamRequest,
+        send_result: &Result<ClientResponse, UpstreamRequestError>,
+    ) {
+        let status_code: StatusCode;
+        let mut sc = "-";
+        let result: &str;
+        match send_result {
```

Comment:
_nit_ Assign this in a tuple, similar to this snippet:

```
let (code, result) = match send_result {
  Ok(ref response) => (response.status().as_str(), "success"),
  Err(_) => ("-", "some_failure"),
};
```

Reply:
cool, I'll do that, thanks